### ISSUE TITLE
external_id missing UpdateOrderInput type

### ISSUE THREAD

**Issue by yash-madaan** [medusajs/medusa#4388]
### Describe the bug

Hi Team, 
I want to update external_id after creating the order, 
But there is no field declared in UpdateOrderInput type
Can you please help me to resolve this ?

### System information

Medusa version (including plugins): 1.8.0
Node.js version:v16.13.1
Database: Postgres
Operating system: Mac OS
Browser (if relevant):

**Comment by yash-madaan** [comment]
Hi @olivermrbl , please help me out in this.

**Comment by Abhijeetmishr** [comment]
Hi @olivermrbl,
I am interested in contributing to this project.
Any suggestions regarding project setup and bug I should look out for before starting to contribute.
Regards,
Abhijeet

**Comment by sradevski** [comment]
Hey, thanks for the report! Since v2 brought a lot of architectural and API changes on the backend, we will be closing this ticket since it no longer applies to our new setup, or the issue has already been fixed. If you are still facing issues with v1,
please open a new ticket and we will address it as soon as possible. Thanks! 🙏
